CharacterCreation/06_Skills.txt

```diff
@@ -90,9 +94,9 @@ are listed below.
 Intimidation:	To strike fear into the hearts of your enemies
 	
 == Intelligence ==
-Hacking:	To pwn computer systems
-Medicine:	To repair life forms
-Repair:	To repair machines
+Hacking [I]:	To pwn computer systems
+Medicine [I]:	To repair life forms

```

Comment:
Let's use the term "organics" or some such, rather than "life forms". Machines count as life under some definitions.

Reply:
I would say biological life forms; you and James wanted to avoid saying "organic" to leave space to develop non-carbon, non-synthetic life.

Reply:
The rules are trending in the direction of standardizing on biological and mechanical character categories. Please don't bring up cyborgs